src/components/button-group/Demo.vue

```diff
@@ -5,25 +5,25 @@
     </div>
 
     <UIButtonGroup :spread="spread" :vertical="vertical">
-        <UIButton category="primary">
+        <UIButton theme="blue">
             Places
         </UIButton>
-        <UIButton category="primary">
+        <UIButton theme="blue">
             List
         </UIButton>
-        <UIButton category="primary">
+        <UIButton theme="blue">
             Map
         </UIButton>
     </UIButtonGroup>
 
     <UIButtonGroup class="mt-10" :spread="spread" :vertical="vertical">
```

Comment:
The focus border on these 3 is wider than the focus border on the top three.

Reply:
it is just the design of the outline buttons 🤷 
if the outline focus border was touching the button it would look like the ouline has just grown.

I think it's a different style of button and this is acceptable, it just might look weird in comparison on the demo website